### ISSUE TITLE
2025.01 Release Tracking

### ISSUE THREAD

**Issue by rtsisyk** [organicmaps/organicmaps#9978]
**Expected Store Submission:** 2025-01-18
**Expected Full Rollout:** 2025-01-23
**Release Manager:** @rtsisyk

Please post status updates as comments on this ticket while progressing through the checklist. Include links and/or screenshots where applicable.

- [x] The data generator is running.
- [x] The release notes are drafted in Etherpad 
- [x] No blockers in [2025.01 milestone](https://github.com/organicmaps/organicmaps/milestone/13).
- [x] A PR with the release notes is raised.
- [x] A PR with the new data is raised - https://github.com/organicmaps/organicmaps/pull/10055
- [ ] QA sign-off:
    - [x] Master has been tested manually on Android by @rtsisyk.
    - [ ] Master has been tested manually on Android by @vng.
    - [x] Master has been tested manually on iOS by @rtsisyk.
    - [ ] Master has been tested manually on iOS by @vng.
    - [ ] Master has been tested by [Android Monkey](../.github/workflows/android-monkey.yaml) on the latest master.
- [x] A PR with the new data has been merged into master.
- [x] A new release `release/2025.01.05` branch has been created from master.
- [x] Metaserver is updated to include new data version
- [x] Data synchronization has been started
- [x] [Android Beta](../.github/workflows/android-beta.yaml) pipeline went through on `release/20XX.MM.DD` branch.
- [x] [iOS Beta](../.github/workflows/ios-beta.yaml) pipeline went through on `release/20XX.MM.DD` branch.
- [x] AppStore metadata, screenshots, and keywords are merged into git.
- [x] "App Store Version Release" in the AppStore draft set to "Manually release this version".
- [x] A PR with the release notes has been merged into master.
- [x] [Android Release](../.github/workflows/android-release.yaml) pipeline went through on `release/20XX.MM.DD` branch.
- [ ] [The iOS App draft in AppStore](https://appstoreconnect.apple.com/apps/1567437057/distribution/ios/version/deliverable) has the same date as the release (`20XX.MM.DD`).
- [x] An iOS build from the iOS Beta pipeline has been submitted to review to "Public" testers in [TestFlight](https://appstoreconnect.apple.com/teams/92fab819-8b15-4d04-bd75-2bd32b523c3f/apps/1567437057/testflight).
- [x] "Build" in the AppStore draft set to the build created by the iOS Beta pipeline.
- [x] [iOS Release](../.github/workflows/ios-release.yaml) pipeline went through on `release/20XX.MM.DD` branch - **only metadata, without screenshots**.
- [x] "Phased Release for App Store Automatic Updates" in the AppStore draft set to "Release update over 7-day period using phased release".
- [x] AppStore (PROD) has been sent to review.
- [x] Android signed APK from Google Play has been uploaded to the GitHub Release as `OrganicMaps-YYMMDDNN-google-release.apk`.
- [x] Android Web (`OrganicMaps-YYMMDDNN-web-release.apk`) and Google Play (`OrganicMaps-YYMMDDNN-google-release.apk`) APKs are shared in Telegram with the release notes.
- [x] Google Play Production sent for a review with automatic <s>10%</s>1% phased rollout
- [x] Data is synchronized
- [x] iOS RC TestFlight is approved
- [x] iOS RC TestFlight link is published on Telegram with the release notes.
- [x] F-Droid [knows about the new build](https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/app.organicmaps.yml).
- [x] Google Play Closed Beta is approved and published via Managed Publishing to 100%.
- [x] Google Play Managed Publishing Check
- [x] Android RC is sent for review to Huawei AppGallery with manual release (TBD)
- [x] Metaserver is updated to include new app versions 
- [x] Google Play 20% phased rollout
- [x] The release final sign-off
	- [x] Firebase reports by TBD
	- [x] Google Play reports by TBD
	- [ ] TestFlight reports by TBD
	- [ ] Email & Telegram reports by TBD
- [x] AppStore 100% rollout
- [x] Google Play 100% rollout
- [x] App Gallery 100% rollout
- [x] GitHub - remove "pre-release" and mark the release as "latest"
- [x] F-Droid build status check
- [x] The release branch is merged into master
- [ ] News Publishing
  	- [x] Publish Telegram news post
  	- [ ] Publish the announcement on Fosstodon
  	- [ ] Publish the announcement on X (Twitter)
  	- [ ] Publish news on the website
  	- [ ] Publish the announcement on Facebook
  	- [ ] Publish the announcement on Instagram
  	- [ ] Publish the announcement on Reddit
  	- [ ] Publish the announcement on LinkedIn

**Comment by rtsisyk** [comment]
Who is going to take this on? I am planning to publish updated documentation, but it would be great if someone else could validate it.

**Comment by rtsisyk** [comment]
@pastk do you want to practice running the map generator?

**Comment by pastk** [comment]
> @pastk do you want to practice running the map generator? Let's start it tomorrow, on Janurary 1. Data will be ready by Jan 5.

Yeap! I'll need map gen server access and perhaps instructions (or hope that everything is setup there).

**Comment by rtsisyk** [comment]
We are currently blocked on the map generation. Please join #data on Zulip to discuss.

**Comment by rtsisyk** [comment]
I've optimistically created https://github.com/organicmaps/organicmaps/tree/release/2025.01.12. Let's start testing it to see if it ready for the release.

**Comment by rtsisyk** [comment]
Please help with preparing the release notes: https://etherpad.wikimedia.org/p/OM-release_notes

**Comment by pastk** [comment]
> Please help with preparing the release notes: https://etherpad.wikimedia.org/p/OM-release_notes

Updated!

The automatically generated relnotes were quite bad, major features were missed like a speed limit sign and massive iOS track recording improvements. With such shortcomings its better not to auto-generate at all, as someone will have to re-scan commits/PRs anyway.

**Comment by pastk** [comment]
- https://github.com/organicmaps/organicmaps/pull/10074

**Comment by rtsisyk** [comment]
TestFlight is on review

**Comment by rtsisyk** [comment]
Firebase App Distribution - sent

**Comment by rtsisyk** [comment]
Telegram: https://t.me/OrganicMapsApp/434

**Comment by rtsisyk** [comment]
Do we want to run generator one more time to have new data @vng @pastk ?

**Comment by pastk** [comment]
Yeap a new data would be great!

**Comment by VoxelPrismatic** [comment]
Should we go ahead and recapture the maps so it's up to date with Jan20 instead of Jan05?

**Comment by pastk** [comment]
I've started new maps generation (data as of Jan 21).

**Comment by pastk** [comment]
New data is ready
- https://github.com/organicmaps/organicmaps/pull/10126

**Comment by pastk** [comment]
The data and the release notes are ready.

@rtsisyk let's push a RC?
And I think we can tag the new release already so that F-Droid picks it up.

**Comment by rtsisyk** [comment]
I've rebased the release branch after merging #10104 and #10108.

**Comment by rtsisyk** [comment]
250121 data sync is in progress.

**Comment by pastk** [comment]
> I've rebased the release branch after merging #10104 and #10108. 

Then we need to update the release notes and their translations and allow some time for beta-testing again..

**Comment by rtsisyk** [comment]
> > I've rebased the release branch after merging [#10104](https://github.com/organicmaps/organicmaps/pull/10104) and [#10108](https://github.com/organicmaps/organicmaps/pull/10108).
> 
> Then we need to update the release notes and their translations and allow some time for beta-testing again..

I've rebased it with your commits related to the release notes. Please check.

**Comment by rtsisyk** [comment]
https://github.com/organicmaps/organicmaps/commits/release/2025.01.24/

**Comment by rtsisyk** [comment]
We have an issue with AppStore screenshots again...

```
[12:08:25]: 🚫 Invalid screenshots were detected! Here are the reasons:
[12:08:25]: 🚫 Error: ../screenshots/ios/de-DE/0_APP_IPAD_PRO_3GEN_129_0.jpg - Invalid screen size (Actual size is 2064x2752. See the specifications to fix https://help.apple.com/app-store-connect/#/devd274dd925)
[12:08:25]: 🚫 Error: ../screenshots/ios/de-DE/1_APP_IPAD_PRO_3GEN_129_1.jpg - Invalid screen size (Actual size is 2064x2752. See the specifications to fix https://help.apple.com/app-store-connect/#/devd274dd925)
[12:08:25]: 🚫 Error: ../screenshots/ios/de-DE/2_APP_IPAD_PRO_3GEN_129_2.jpg - Invalid screen size (Actual size is 2064x2752. See the specifications to fix https://help.apple.com/app-store-connect/#/devd274dd925)
[12:08:25]: 🚫 Error: ../screenshots/ios/de-DE/3_APP_IPAD_PRO_3GEN_129_3.jpg - Invalid screen size (Actual size is 2064x2752. See the specifications to fix https://help.apple.com/app-store-connect/#/devd274dd925)
[12:08:25]: 🚫 Error: ../screenshots/ios/hu/0_APP_IPAD_PRO_3GEN_129_0.jpg - Invalid screen size (Actual size is 2064x2752. See the specifications to fix https://help.apple.com/app-store-connect/#/devd274dd925)
[12:08:25]: 🚫 Error: ../screenshots/ios/hu/1_APP_IPAD_PRO_3GEN_129_1.jpg - Invalid screen size (Actual size is 2064x2752. See the specifications to fix https://help.apple.com/app-store-connect/#/devd274dd925)
[12:08:25]: 🚫 Error: ../screenshots/ios/hu/2_APP_IPAD_PRO_3GEN_129_2.jpg - Invalid screen size (Actual size is 2064x2752. See the specifications to fix https://help.apple.com/app-store-connect/#/devd274dd925)
[12:08:25]: 🚫 Error: ../screenshots/ios/hu/3_APP_IPAD_PRO_3GEN_129_3.jpg - Invalid screen size (Actual size is 2064x2752. See the specifications to fix https://help.apple.com/app-store-connect/#/devd274dd925)
[12:08:25]: 🚫 Error: ../screenshots/ios/hu/4_APP_IPAD_PRO_3GEN_129_4.jpg - Invalid screen size (Actual size is 2064x2752. See the specifications to fix https://help.apple.com/app-store-connect/#/devd274dd925)
```

**Comment by pastk** [comment]
> I've rebased it with your commits related to the release notes. Please check.

The merged relnotes don't contain changes added after Jan 8. They contain only changes that were included into the last beta.

**Comment by rtsisyk** [comment]
AppStore, Google Play, Huawei on review.

**Comment by rtsisyk** [comment]
> [The iOS App draft in AppStore](https://appstoreconnect.apple.com/apps/1567437057/distribution/ios/version/deliverable) has the same date as the release (20XX.MM.DD).

This item requires clarification. The tag was created in advance, on 24th. The build was started on 26th. AppStore now has 2025.01.26.

**Comment by rtsisyk** [comment]
https://github.com/organicmaps/organicmaps/issues/10140

**Comment by rtsisyk** [comment]
I've recorded videos for all three background features (navigation, track recorder, Android Auto) and re-submitted the app for Google Play review.

**Comment by rtsisyk** [comment]
AppStore has been approved.

<img width="1173" alt="Image" src="https://github.com/user-attachments/assets/6bc21d1f-e84e-4084-8af0-8edaa34d393e" />

@kirylkaveryn could you please check TestFlight/AppStore reports?

**Comment by pastk** [comment]
F-Droid build and AF changes (NonFreeNet removed) have been merged.
https://gitlab.com/fdroid/fdroiddata/-/merge_requests/19017

**Comment by rtsisyk** [comment]
Google Play Closed Beta has been approved. I've pushed the same bundle for a review for Production.

**Comment by rtsisyk** [comment]
Google Play Production - 20%.

**Comment by pastk** [comment]
Published on F-Droid!

**Comment by pastk** [comment]
> Google Play Production - 20%.

Should we move forward if everything is ok?

**Comment by rtsisyk** [comment]
@oleg-rswll could you or Ulyana please post an update about this release on social media?

**Comment by rtsisyk** [comment]
Data-only:

https://github.com/organicmaps/organicmaps/actions/runs/13373604120
https://github.com/organicmaps/organicmaps/actions/runs/13373599539
https://github.com/organicmaps/organicmaps/tree/release/2025.01.12

CC: @pastk

**Comment by pastk** [comment]
> https://github.com/organicmaps/organicmaps/tree/release/2025.01.12

2025.01.24
